### ISSUE TITLE
Training dataset

### ISSUE THREAD

**Issue by chouchonkit** [chavinlo/TempoFunk#2]
Hi，I wonder what is the text for dataset?

**Comment by chavinlo** [comment]
text? what do you mean?

If you mean what dataset we used, we scrapped a small tiktok account and used the label "dancing coreography": https://www.tiktok.com/@dancetutorials.tv

We are currently using a small chunk of ShutterStock

**Comment by chouchonkit** [comment]
thanks for your reply! I mean what is "/workspace/disk/dataset/processed/val/text" from the sample_training.py. 
I read about the paper "MAKE-A-VIDEO: TEXT-TO-VIDEO GENERATION WITHOUT TEXT-VIDEO DATA" and the paper use unlabeled video data training on the training part, I'm not understand why need to use text for training.
Could you explain a little bit more what is the difference between your code and the paper method? 
Thank you so much!
